packages/orchestration/src/examples/staking-combinations.contract.js

```diff
@@ -128,8 +118,15 @@ const contract = async (
     StakingCombinationsInvitationMakersI,
   );
 
+  const { makeLocalAccount } = orchestrateAll(sharedFlows, {});
+  // XXX sharedLocalAccountP expects a Promise but this is a vow https://github.com/Agoric/agoric-sdk/issues/9822
+  /** @type {any} */
```

Comment:
Since this is an example contract, consider over communicating what's going on here:

```suggestion
  /** 
   * Setup a shared local account for use in async-flow functions.
   * Typically, exo initState functions need to resolve synchronously, but 
   * `makeOnce` allows us to provide a Promise. When using this inside a flow,
   * we must await it to ensure the account is available for use.
   * @type {unknown}
   */
```